pkg/apiserver/server.go

```diff
@@ -49,6 +50,11 @@ var (
 func serverRegister(r *gin.Engine, conf *config.ChaosDashboardConfig) {
 	listenAddr := fmt.Sprintf("%s:%d", conf.ListenHost, conf.ListenPort)
 
+	if conf.EnablePprof {
```

Comment:
As I commented above, maybe we need to move the position of "pprof.Register(r)".

Reply:
At the first time, I did this, but I found that we couldn't consume the `conf.EnablePprof` such flag anymore, right?

Reply:
Oh oh, let me take a look.

Reply:
Maybe you can do something like below :
`
func newEngine (conf *config.ChaosDashboardConfig) *gin.Engine {
	r := gin.Default()

	if conf.EnablePprof {
		// default is "debug/pprof"
		pprof.Register(r)
	}
        ....................
}
`